pipeline/enigma/enigma-processing.py

```diff
@@ -100,6 +103,23 @@ def main():
     f_in.close()
     f_out.close()
 
+def cleanup_HGVS(reference_sequence, cdna_hgvs, hp, evm):
+    """The pyhgvs library breaks on HGVS strings that do not specify a ref or alt allele.  The hgvs
+    library is robust to these HGVS strings, but does not map to hg38.  Here. use the HGVS library
+    to convert the HGVS cDNA string to a HGVS genomic string (against GRCh37), and convert that genomic
+    string back to a HGVS cDNA string.  The resulting string can be parsed by the pyhgvs library more
+    reliably.  But, it has a strange error with certain insertions.  So don't do this with insertions..."""
+    if re.search("ins", cdna_hgvs) and not re.search("del", cdna_hgvs):
+        return reference_sequence + ":" + cdna_hgvs
```

Comment:
If this sets `items[column_idx["HGVS"]]` by returning `reference_sequence + ":" + cdna_hgvs`, wouldn't line 87 `HGVS_cDNA = items[column_idx["Reference_sequence"]] + ":" + items[column_idx["HGVS"]]` come out with a double reference sequence?